martin/src/config/file/tiles/postgres/resolver/query_tables.rs

```diff
@@ -36,6 +39,15 @@ pub async fn query_available_tables(pool: &PostgresPool) -> PostgresResult<SqlTa
     for row in &rows {
         let schema: String = row.get("schema");
         let table: String = row.get("name");
+
+        // If a list of configured tables was provided to the function, because auto_publish set
+        // to false, skip processing the tables that are not in the list.
```

Comment:
nit: I think this can be more explicit

```suggestion
        // If auto_publish is not active, a user can provide a set of tables.
        // The query above might find more tables than the statically configured ones.
        // Therefore we filter to limit us to the subset of available and configured tables
```

Reply:
This doesn't read right to me. We wouldn't restrict to the tables found by the query, but rather the tables in the config.

Reply:
Fair. I have updated the phrasing, calling the subset-behaviour out is nice as the next generation of devs might not notice this otherwise

Reply:
> a user can provide a set of tables.

In a way I suppose, through defining them in the config.yml and setting auto_publish to false.

How about:
```
// Within the config, if auto_publish is false or omitted, the list of schema and table names set explicitly 
// under the tables key is provided to the function. As the query above may return more tables than 
// explicitly defined, these are filtered out below.
```

But since this comment should be first and foremost to help people who didn't author this bit of code, I'll leave it to your discretion on what makes the most sense.

Reply:
Sure, your version is better

Reply:
Sure, your version is better